### PR TITLE
[plugin-web-app] Add steps to check and set focus on elements by locator

### PR DESCRIPTION
**Pull request by valfirst**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 1]
```diff
@@ -332,6 +349,26 @@ Then context element is in focus
...
+----
+Then element located by `$locator` is $focusState
+----
+* `$locator` - The <<_locator,locator>> to find an element.
+* `$focusState` - The state to verify: `in focus` or `not in focus`.
+
+.Check the element with ID "input-username" is in focus and do keyboard input
+[source,gherkin]
+----
+Then element located by `id(input-username)` is $focusState
```
Comment (uarlouski) [inline docs/modules/plugins/pages/plugin-web-app.adoc:367]: $focusState -> focus?

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/FocusSteps.java [hunk 2]
```diff
@@ -43,23 +48,54 @@ public FocusSteps(IUiContext uiContext, WebJavascriptActions javascriptActions,
...
+    /**
+     * Sets the focus on the element found by the specified locator, if it can be focused. The focused element is the
+     * element that will receive keyboard and similar events by default.
+     *
+     * @param locator The locator to find an element.
+     */
+    @When("I set focus on element located by `$locator`")
+    public void setFocusOnElement(Locator locator)
+    {
+        baseValidations.assertElementExists("Element to focus on", locator).ifPresent(this::setFocusOnElement);
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/FocusSteps.java:65]: I think "Element to set focus on" suits better the focus steps wording

File: docs/modules/plugins/pages/plugin-web-app.adoc (hunk 1 above)
Comment (uarlouski) [reply docs/modules/plugins/pages/plugin-web-app.adoc:367]: okay, I just thought we use real values in examples, not parameter refs

**Review by uarlouski — APPROVED** [review]

File: docs/modules/plugins/pages/plugin-web-app.adoc (hunk 1 above)
Comment (valfirst) [reply docs/modules/plugins/pages/plugin-web-app.adoc:367]: oh, my bad, I didn't realized this is an example

**Pull request merged**